app/build.gradle

```diff
@@ -35,14 +35,16 @@ android {
         }
     }
     buildFeatures {
+        //noinspection DataBindingWithoutKapt
```

Comment:
can we remove this line since it's commented out?

Reply:
I got your message about the Android Studio warning, to fix this, can you add the kotlin-kapt plugin to the top of this file?

See example in the Cupcake app gradle file:
https://github.com/google-developer-training/android-basics-kotlin-cupcake-app/blob/main/app/build.gradle#L4

You can remove lines 17 and 18. And then add this instead: 

plugins {
    id 'com.android.application'
    id 'kotlin-android'
    id 'kotlin-kapt'
}